compress_zarr.py

```diff
@@ -41,12 +41,12 @@ def blosc_compressor_lib(trunc_bits, chunk_factor):
     return opts
 
 def lossless_compressor_lib(trunc_bits, chunk_factor):
-    clevels = [ 1, 3, 5, 9 ]
+    clevels = [ 1, 3, 5, 7, 9 ]
 
     opts = []
     for clevel, tb, cf in itertools.product(clevels, trunc_bits, chunk_factor):
         opts.append({
-            'name': 'zlib',
+            'name': 'lossless-zlib',
```

Comment:
this is some confusion I introduced earlier.  these aren't actually lossless when truncation is enable.  I don't know that it's worth changing that now, but if you have ideas feel for how to clear the naming up up go for it.

Reply:
Hmm, maybe "numcodecs-zlib" or the like?  KLB also supports zlib and bzip2, so ideally it would be able to distinguish those